src/data/faq.json

```diff
@@ -573,6 +573,15 @@
                             "For more information on the STIKO vaccination recommendations and all about COVID-19 and vaccination, see <a href='https://www.rki.de/DE/Content/Infekt/Impfen/ImpfungenAZ/COVID-19/COVID-19.html' target='_blank' rel='noopener noreferrer'>RKI - Impfungen A - Z - COVID-19 und Impfen (German)</a>."
                         ]
                     },
+                    {
+                        "title": "2/1 vaccination certificates",
+                        "anchor": "vc_2of1",
+                        "active": false,
+                        "textblock": [
+                            "The Android app and iOS app versions 2.17.1 differ with regard to the display of the 2/1 vaccination certificate after a first vaccination with Johnson and Johnson: In the iOS app, the complete vaccination status is currently displayed immediately. For the Android app, this is currently not the case until day 15. We will immediately adjust this with the Android app so that immediate full immunization is also displayed here with a 2/1 vaccination certificate.",

```

Comment:
"We will immediately adjust this with the Android app so that immediate full immunization is also displayed here with a 2/1 vaccination certificate."

"immediately adjust" would mean in the next second or two

instead

"We are urgently in the process of adjusting ..."

also the word order  in "immediate full immunization" could be improved, so I suggest the full changed sentence as:

**"We are urgently in the process of adjusting this with the Android app so that full immunization is also immediately displayed here with a 2/1 vaccination certificate."**

Reply:
alright, thank you!